python/hsfs/util.py

```diff
@@ -139,6 +140,26 @@ def get_hudi_datestr_from_timestamp(timestamp):
     return datetime.fromtimestamp(timestamp / 1000).strftime("%Y%m%d%H%M%S%f")[:-3]
 
 
+def convert_event_time_to_timestamp(event_time, time_zone=timezone.utc):
+    if event_time is None:
+        return None
+    if isinstance(event_time, str):
+        if event_time:
+            return get_timestamp_from_date_string(event_time, time_zone)
+    if isinstance(event_time, pd._libs.tslibs.timestamps.Timestamp):
+        # pandas.timestamp represents millisecond in decimal
```

Comment:
This comments sounds a bit misleading. pd.Timestamp.timestamp() returns unix epoch time in seconds and we multiply by 1000 because we want to return unix epoch time in milliseconds.